src/Xhgui/Saver.php

```diff
@@ -21,9 +21,9 @@ public static function factory($config)
             case 'mongodb':
             default:
                 $mongo = new MongoClient($config['db.host'], $config['db.options']);
-                $mongo->{$config['db.db']}->results->findOne();
-                $profiles = new Xhgui_Profiles($mongo->{$config['db.db']});
-                return new Xhgui_Saver_Mongo($profiles);
+                $collection = $mongo->{$config['db.do']}->results;
+                $collection->findOne();
```

Comment:
I don't know if this `findOne()` call was necessary, but I kept it to be on the safe side.

Reply:
I think its needed to force mongo to connect.

Reply:
I wonder why it's necessary to force Mongo to connect?

because in typical use cases when using register shutdown function, can't see errors so the connection is made earlier?